commands/chatgpt.py

```diff
@@ -13,7 +13,7 @@ def execute(self, message):
         self.bot.set_state(message.chat.id, ChatBotStates.WAITING_FOR_CHATGPT_INPUT)
 
 
-def register_chatgpt_handlers(bot):
+def New ПРregister_chatgpt_handlers(bot):
     """
     Register the state handler for ChatGPT input.
     """
```

Comment:
You can increase tokens for input   max_tokens=50 , my suggestion is increasing it form 50 to 128 characters